### PR TITLE
Images opacity fixed

### PR DESCRIPTION
**Pull request by decarbonite**
# Purpose / Goal
This is a fix for issue #62 
It also fixed another problem where the opacity changes to full opacity "1.0" when you click on the image in the slidebar in the bottom regardless of the current opacity selection.

# Type

Bug Fix

* [x] Bug Fix
* [ ] Refactoring / Technology upgrade
* [ ] New Feature
* [ ] Documentation
* [ ] Other : | Please Specify |

**Note** : Please ensure that you've read contribution [guidelines](CONTRIBUTING.md) before raising this PR.

### CODE REVIEW

**Comment by amitguptagwl** [comment]
Thanks for the PR. But this fix seems error prone. Imagine the situation when light up tool is not loaded.

Instead you should have assigned the value to some global variable and you should that value to set image opacity. Moreover, as it seems a configuration, you may set this variable in appConfig in settings.js.

> Join [Team](https://github.com/socialat/team/issues/1) to support opensource community.
> [Bookmark](https://github.com/NaturalIntelligence/imglab/stargazers) this project.

File: tags/actions/lightbulb-action.tag.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <action-lightbulb>
     <div class="range-slider">
-        <input class="input-range" type="range" step="0.1" value="1.0" min="0" max="1.0" onchange={changeFillOpacity}>
+        <input id="input-range" type="range" step="0.1" value="1.0" min="0" max="1.0" onchange={changeFillOpacity}>
```
Comment (amitguptagwl) [inline tags/actions/lightbulb-action.tag.html]: This change is not required. just set global property in changeFillOpacity().

File: tags/images-slider.tag.html [hunk 2]
```diff
@@ -169,6 +169,22 @@
         function loadIntoWorkArea(e){
             imgSelected = tag.thumbnails[e.item.index];
             riot.mount("workarea",{ img : imgSelected });
+            readyToOpacity()
```
Comment (amitguptagwl) [inline tags/images-slider.tag.html]: This file is not the correct place. You should do the change in `workarea.tag.html`. In addition, a component will be loaded only page is loaded so you need not to check `document.readyState`

**Comment by amitguptagwl** [comment]
Same issue with the latest changes. You're expecting that opacity slider will always be present which is wrong.

**Comment by decarbonite** [comment]
I think I don't understand the issue completely, maybe an answer to this question would help me clarify things. 

What should be the behavior of the program if the light tool was not loaded? 

What my code does is that it sets opacity depending on the value of the slider and the slider can only be shown if the light tool is clicked.

 Correct me if I'm wrong, the problem with my code is that I'm getting the value of opacity from the slider and storing it into a variable without making sure that the slider even exit, so it could end up as a garbage value?

**Comment by amitguptagwl** [comment]
An image can have different colors. Many times SVG shapes we draw are not completely visible due to the colors in image. Hence we want to change the opacity of the image. 

The one way of doing this is to change the opacity for every image when it loads (we're already doing it) or do it once for all the image like application settings.

Slider is just the medium to set the value. When you click on lighup tool a slider loads in action bar. Tomorrow we may implement another feature which loads some other action. It means, opacity slider will be unloaded.

Since your all the solutions are tightly coupled with the presence of opacity slider, they're incorrect. I've just fixed this issue. Please check this [commit](https://github.com/NaturalIntelligence/imglab/commit/db8b3ea480ef263f73bb91711d6fa31f93262c06) for learning purpose.

Feel free to clear your doubts. This is the whole purpose of creating first timers friendly issues.

**Pull request closed without merging**